Gruntfile.js

```diff
@@ -186,61 +191,61 @@ module.exports = grunt => {
                 expand: true,
                 src: [ '*/' ],
                 include: [ '**/translation.json', '**/translation-additions.json' ],
-                rename( dest, src ) {
+                rename(dest, src) {
                     return `${dest + src}translation-combined.json`;
                 },
                 dest: 'locales/build/'
             }
         }
-    } );
+    });
 
-    grunt.registerTask( 'system-sass-variables', 'Creating _system_variables.scss', () => {
+    grunt.registerTask('system-sass-variables', 'Creating _system_variables.scss', () => {
         const SYSTEM_SASS_VARIABLES_PATH = 'app/views/styles/component/_system_variables.scss';
-        const config = require( './app/models/config-model' );
-        grunt.file.write( SYSTEM_SASS_VARIABLES_PATH, `$base-path: "${config.server[ 'base path' ]}";` );
-        grunt.log.writeln( `File ${SYSTEM_SASS_VARIABLES_PATH} created` );
-    } );
+        const config = require('./app/models/config-model');
+        grunt.file.write(SYSTEM_SASS_VARIABLES_PATH, `$base-path: "${config.server[ 'base path' ]}";`);
```

Comment:
I don't think it makes sense to change the convention for parents without also changing it for braces (`[` and `]`). I'd tend to do those in the same commit. Any reason not to?